wdfReader.py

```diff
@@ -0,0 +1,285 @@
+from __future__ import print_function
+import struct
+import numpy
+
+l_int16 = 2
```

Comment:
Here you could use the numpy data types and their attributes (itemsize, etc.) for more readability. See https://docs.scipy.org/doc/numpy-1.15.1/reference/arrays.dtypes.html